crates/forge/bin/cmd/test/mod.rs

```diff
@@ -81,6 +81,14 @@ pub struct TestArgs {
     #[arg(long, value_name = "TEST_FUNCTION")]
     debug: Option<Regex>,
 
+    /// Generate flamegraph if a single test is executed.
```

Comment:
```suggestion
    /// Generate a flamegraph for a single test. Implies `--decode-internal`.
```

Please also apply to `flamechart`